app/generated_resources.grd

```diff
@@ -10935,6 +10935,9 @@ Please help our engineers fix this problem. Tell us what happened right before y
     <message name="IDS_SHOW_KALEIDOSCOPE" desc="Menu item to launch the Brave watch feature">
       Watch
     </message>
+    <message name="IDS_EXTENSION_PROMPT_WARNING_IPFS" desc="Permisson string for access to ipfs api.">
+      Publish and retrieve files from IPFS
+    </message>
```

Comment:
@spylogsster: sorry, didn't notice during the review, but this needs to go into `app/brave_generated_resources.grd`. We don't add our strings to chromium GRDs.

Reply:
fixed here https://github.com/brave/brave-core/pull/8409